src/features/classification/security-controls/__tests__/utils.test.js

```diff
@@ -18,77 +18,54 @@ const { COLLAB_ONLY, COLLAB_AND_COMPANY_ONLY, PUBLIC } = SHARED_LINK_ACCESS_LEVE
-            accessPolicy = { externalCollab: {}, download: {}, app: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortAllRestrictions);
-        });
-
-        test('should return correct message when download and either shared link, or external collab restrictions are present', () => {
-            accessPolicy = { sharedLink: {}, download: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortSharingDownload);
-            accessPolicy = { externalCollab: {}, download: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortSharingDownload);
-        });
-
-        test('should return correct message when app and either shared link, or external collab restrictions are present', () => {
-            accessPolicy = { sharedLink: {}, app: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortSharingApp);
-            accessPolicy = { externalCollab: {}, app: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortSharingApp);
-        });
-
-        test('should return correct message when app and download restrictions are present', () => {
-            accessPolicy = { download: {}, app: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortDownloadApp);
-        });
-
-        test('should return correct message when there are shared link or external collab restrictions', () => {
-            accessPolicy = { sharedLink: {}, externalCollab: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortSharing);
-
-            accessPolicy = { sharedLink: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortSharing);
-
-            accessPolicy = { externalCollab: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortSharing);
-        });
-
-        test('should return correct message when there is a download restriction', () => {
-            accessPolicy = { download: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortDownload);
-        });
-
-        test('should return correct message when there is a download restriction', () => {
-            accessPolicy = { app: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortApp);
-        });
-
-        test('should return correct message when there is a watermark restriction', () => {
-            accessPolicy = { watermark: {} };
-            expect(getShortSecurityControlsMessage(accessPolicy)[0].message).toBe(messages.shortWatermarking);
+        test.each`
```

Comment:
Refactored this into a table which should make it easier to extend. All previous test cases are covered.